apps/web/pages/booking/[uid].tsx

```diff
@@ -128,7 +128,8 @@ export default function Success(props: SuccessProps) {
   )?.videoCallUrl;
 
   const status = props.bookingInfo?.status;
-  const reschedule = props.bookingInfo.status === BookingStatus.ACCEPTED;
+  const reschedule =
```

Comment:
I don't know why this variable's name is reschedule. Could we rename it better? or maybe see how it's used exactly and improve the logic.

Reply:
reschedule is a bool variable which checks the status of reschedule. It is later user to check if a valid reschedule request was made. It is valid, if the booking status was either pending or accepted.
```const reschedule=props.bookingInfo.status === BookingStatus.ACCEPTED || props.bookingInfo.status === BookingStatus.PENDING;```
reschedule would be true if the booking status is ```Accepted``` or ```Pending```.
Based on this, is there any improvisation?